Make.inc

```diff
@@ -144,6 +147,13 @@ CFLAGS_add += -I$(OPENLIBM_HOME)/ld128
 endif
 endif
 
+ifneq ($(filter $(ARCH),i387 amd64),)
+LONG_DOUBLE_NOT_DOUBLE := 1
```

Comment:
Perhaps add a comment about what this means?

Reply:
It determines whether `long double` is `double` or not

Reply:
```suggestion
# Determines whether `long double` is the same as `double` on this arch.
# linux x86_64, for instance, `long double` is 128 bits wide, whereas on macOS aarch64,
# `long double` is the same as `double`. 
LONG_DOUBLE_NOT_DOUBLE := 1
```

Reply:
80 bits on x86_64 linux, but otherwise sounds good ;)

Reply:
unless you're talking about aarch64 for the linux case also

Reply:
Ah, I can just edit your comment and then apply it